articles/install-guide/csinstall.md

```diff
@@ -135,7 +136,86 @@ Of course, you can also build and run Q# programs from the command line by simpl
 
         You should see the following output: `Hello quantum world!`
 
+## Q# with other .NET languages <a name="dotnet"></a>
+
+Q# is compiled to C# for the purposes of local simulation. This makes them very easy to combine. However, you can also use other .NET languages like F# or VB\.NET, although the process is less straightforward.
```

Comment:
Is the slash in `VB\.NET` an escape character or a typo?

Reply:
It's just to avoid the builder compiling it as a hyperlink.

Reply:
Do we need to do the same for other occurrences of VB.NET as well then?